### PR TITLE
.coafile: Add YAMLLintBear

### PR DESCRIPTION
**Pull request by atultherajput**
Add YAMLLintBear for fixing issues with .yml files.

Closes https://github.com/coala/projects/issues/313

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [X] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by sils** [comment]
Please address issues by gitmate-bot.

**Comment by atultherajput** [comment]
@sils 
I had modified ".travis.yml" and "_config.yml" files as per gitmate-bot in latest commit but still gitmate-bot is generating same errors.

File: .coafile [hunk 1]
```diff
@@ -25,3 +25,7 @@ bears = HTMLLintBear
 files = **.html
 htmllint_ignore = optional_tag, protocol, indentation,  concerns_separation, capitalization
 ignore = resources/vendors/**, _site/**
+
+[yml]
+bears = YAMLLintBear
+files = **.yml
```
Comment (sils) [inline .coafile]: Hi,

it looks like you're missing a railing newline before the end of the file here. [This article](http://stackoverflow.com/a/5813359/3212182) has more information about this issue.

File: _config.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 title: coala Projects
 license: GNU AFFERO GENERAL PUBLIC LICENSE V3.0
 organization: coala
-description: A website that showcases interesting projects, using Angular JS. 
-baseurl: "" # the subpath of your site, e.g. /blog
-url: "" # the base hostname & protocol for your site
+description: A website that showcases interesting projects, using Angular JS.
+baseurl: ""   # the subpath of your site, e.g. /blog
+url: ""   # the base hostname & protocol for your site
```
Comment (sils) [inline _config.yml]: too many spaces, only use two spaces before `#` please (also above)

File: .travis.yml [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
 
 # Assume bundler is being used, therefore
 # the `install` step will run `bundle install` by default.
 script: ./ci/cibuild
 
 env:
   global:
-  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up installation of html-proofer
+    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true   # speeds up installation
+# of html-proofer
```
Comment (sils) [inline .travis.yml]: weird breaking of comments, rather do one comment in an own line above the line

**Comment by sils** [comment]
unack ffdfa7b

File: .coafile (hunk 1 above)
Comment (atultherajput) [reply .coafile]: done :)

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/368/commits/a74346158788e721b698e39faf581deaa89b4f76

Nice :)

**Comment by sils** [comment]
@rultor merge

**Pull request merged**